perfmetrics/scripts/continuous_test/ml_tests/tf/setup_scripts/setup_container.sh

```diff
@@ -0,0 +1,157 @@
+#!/bin/bash
+
+# Install go lang
+wget -O go_tar.tar.gz https://go.dev/dl/go1.19.4.linux-amd64.tar.gz
+sudo rm -rf /usr/local/go && tar -xzf go_tar.tar.gz && sudo mv go /usr/local
+export PATH=$PATH:/usr/local/go/bin
+
+# Clone the repo and build gcsfuse
+git clone "https://github.com/GoogleCloudPlatform/gcsfuse.git"
+cd gcsfuse
+git checkout log_rotation
+go build .
+cd -
+
+# Mount the bucket
+echo "Mounting the bucket"
+#gcsfuse/gcsfuse --implicit-dirs --max-conns-per-host 100 --disable-http2 --log-format "text" --log-file log.txt --stackdriver-export-interval 60s ml-models-data-gcsfuse myBucket > /home/output/gcsfuse.out 2> /home/output/gcsfuse.err &
+
+# Install tensorflow model garden library
+pip3 install --user tf-models-official==2.10.0
+
+# Fail building the container image if folder.py is not at expected location.
+if [ -f "/root/.local/lib/python3.7/site-packages/official/core/train_lib.py" ]; then echo "file exists"; else echo "train_lib.py file not present in expected location. Please correct the location. Exiting"; exit 1; fi
+if [ -f "/root/.local/lib/python3.7/site-packages/orbit/controller.py" ]; then echo "file exists"; else echo "controller.py file not present in expected location. Please correct the location. Exiting"; exit 1; fi
+
+# Copying tf util files from bucket
+gsutil -m cp gs://gcsfuse-ml-data/tf_kokoro_test/resnet.py .
```

Comment:
Given resnet.py is written by us, so we can create and add this file to our repository with license as in other files. Also, change the name of file to signify that this is script to run model.